tools/nxfuse/mknxfuse.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
Let's use `#!/usr/bin/env bash` to support MAC.

Reply:
Done.